pkg/server/server.go

```diff
@@ -40,7 +40,8 @@ func New(application application.Application) *RegistrationServer {
 			SkipPaths: []string{"/api/v1/health"}, // disable logging for the /api/v1/health endpoint so that our logs aren't overwhelmed
 			Formatter: func(params gin.LogFormatterParams) string {
 				// custom JSON format
-				return fmt.Sprintf(`{"client-ip":"%s", "ts":"%s", "method":"%s", "path":"%s", "proto":"%s", "status":"%d", "latency":"%s", "user-agent":"%s", "error-message":"%s"}`,
+				return fmt.Sprintf(`{"level":"%s", "client-ip":"%s", "ts":"%s", "method":"%s", "path":"%s", "proto":"%s", "status":"%d", "latency":"%s", "user-agent":"%s", "error-message":"%s"}`+"\n",
```

Comment:
Do we really need this `+"\n"`?

Reply:
yes, otherwise the next log will be printed on the same line :/

Reply:
for example:
```
{"client-ip":"10.131.0.16", "ts":"Tue, 09 Nov 2021 10:17:59 UTC", "method":"POST", "path":"/api/v1/signup", "proto":"HTTP/1.1", "status":"202", "latency":"21.070371ms", "user-agent":"Go-http-client/1.1", "error-message":""}{"client-ip":"10.128.2.4", "ts":"Tue, 09 Nov 2021 10:51:30 UTC", "method":"Get", "path":"/", "proto":"HTTP/1.1", "status":"200", "latency":"411.875µs", "user-agent":"Go-http-client/1.1", "error-message":""}{"level":"info","ts":"2021-11-09T10:52:14.853Z","logger":"registration-service","msg":"user not found, creating a new one","encoded_user_id":"42b12331-7d00-46fa-ae3a-e7948b2600f1","timestamp":"Tue, 09 Nov 2021 10:52:14 +0000","commit":"d6be582","user_id":"42b12331-7d00-46fa-ae3a-e7948b2600f1","username":"returninguser1","req_url":":///api/v1/signup","req_headers":{"Accept-Encoding":["gzip"],"Authorization":"*****","Content-Length":["0"],"Content-Type":["application/json"],"Forwarded":["for=90.39.35.154;host=registration-service-toolchain-host-operator.apps.xcoulon-20211109a.devcluster.openshift.com;proto=https"],"User-Agent":["Go-http-client/1.1"],"X-Forwarded-For":["90.39.35.154"],"X-Forwarded-Host":["registration-service-toolchain-host-operator.apps.xcoulon-20211109a.devcluster.openshift.com"],"X-Forwarded-Port":["443"],"X-Forwarded-Proto":["https"]}}
{"level":"info","ts":"2021-11-09T10:52:14.866Z","logger":"registration-service","msg":"UserSignup 42b12331-7d00-46fa-ae3a-e7948b2600f1 created","timestamp":"Tue, 09 Nov 2021 10:52:14 +0000","commit":"d6be582","user_id":"42b12331-7d00-46fa-ae3a-e7948b2600f1","username":"returninguser1","req_url":":///api/v1/signup","req_headers":{"Accept-Encoding":["gzip"],"Authorization":"*****","Content-Length":["0"],"Content-Type":["application/json"],"Forwarded":["for=90.39.35.154;host=registration-service-toolchain-host-operator.apps.xcoulon-20211109a.devcluster.openshift.com;proto=https"],"User-Agent":["Go-http-client/1.1"],"X-Forwarded-For":["90.39.35.154"],"X-Forwarded-Host":["registration-service-toolchain-host-operator.apps.xcoulon-20211109a.devcluster.openshift.com"],"X-Forwarded-Port":["443"],"X-Forwarded-Proto":["https"]}}
```

Reply:
OK